Mathlib/Tactic/Basic.lean

```diff
@@ -274,3 +274,8 @@ elab (name := clearAuxDecl) "clear_aux_decl" : tactic => withMainContext do
     if ldec.isAuxDecl then
       g ← Meta.tryClear g ldec.fvarId
   replaceMainGoal [g]
+
+register_option push_neg.use_distrib : Bool :=
```

Comment:
Why is this here rather than in `Mathlib/Tactic/PushNeg.lean`?

Reply:
I originally put it there because the option doesn't work in the file in which it is defined, but since `PushNeg.lean` doesn't need to actually set the option, it's fine to put it there after all. I have now changed it.